frontend/src/services/auth.js

```diff
@@ -64,6 +95,16 @@ export async function loginUser(email, password) {
       return null;
     }
   } catch (error) {
+    console.error("Error registering student ID:", error);
     return null;
   }
 }
+
+export async function logoutUser() {
+  await axios.post(
+    "http://localhost:8000/auth/logout",
```

Comment:
same reason as for frontend.
```suggestion
    `${process.env.BACKEND_URL}/auth/logout`,
```

Reply:
To keep testing & development easier, I suggest configuring these later